amy/templates/dashboard/training_progress.html

```diff
@@ -75,38 +75,48 @@ <h2>Your progress towards becoming a Carpentries Instructor</h2>
       </td>
     </tr>
     <tr>
-      <th rowspan="2">2. Get Involved</th>
+      <th>2. Get Involved</th>
       <td>
         {% if progress_get_involved %}
           {% include 'includes/progress_trainee_view.html' with progress=progress_get_involved %}
           {% if progress_get_involved.state == "n" %}
-          <p>Please wait while we evaluate your submission.</p>
+          <p>Your submission will be evaluated within 7-10 days.</p>
+          <p>You may edit or delete your submission before it is evaluated. If you delete it, you will be allowed to create a new submission afterwards.</p>
+          {% endif %}
+          <p>Details of your submission are below.</p>
+          <p>
+            <strong>Activity:</strong> {{ progress_get_involved.involvement_type }}<br/>
+            <strong>Date:</strong> {% firstof progress_get_involved.date|date:'F j, Y' "No date provided" %}<br/>
```

Comment:
Nice, I didn't know about `firstof`. How does particular tag work when `progress_get_involved.date` is empty (e.g. `None`)? Does `|date` filter complain?

Reply:
`date` does not complain in that case - I think it returns an empty string. So then `firstof` falls back on `"No date provided"`. It's tested in `test_get_involved_details_not_provided`